client/pkg/logutil/zap.go

```diff
@@ -72,37 +72,21 @@ var DefaultZapLoggerConfig = zap.Config{
 
 // MergeOutputPaths merges logging output paths, resolving conflicts.
 func MergeOutputPaths(cfg zap.Config) zap.Config {
-	outputs := make(map[string]struct{})
-	for _, v := range cfg.OutputPaths {
-		outputs[v] = struct{}{}
-	}
-	outputSlice := make([]string, 0)
-	if _, ok := outputs["/dev/null"]; ok {
-		// "/dev/null" to discard all
-		outputSlice = []string{"/dev/null"}
-	} else {
-		for k := range outputs {
-			outputSlice = append(outputSlice, k)
+	mergePaths := func(old []string) []string {
```

Comment:
Any reason for anonymous function? Would a normal named function be more readable?